chef/cookbooks/crowbar/templates/default/crowbar-pre-upgrade.sh.erb

```diff
@@ -57,12 +57,15 @@ wicked ifreload all
 
 <% if @compute_node %>
 
+# FIXME: remove this once we can install Cloud6 from non-devel repositories
+zypper --non-interactive install crudini
+crudini --set /etc/zypp/zypp.conf main solver.allowVendorChange true
```

Comment:
I'd really prefer to not merge this. Would it be possible to add this to mkcloud/qa-crowbarsetup?